test/format/ops.c

```diff
@@ -992,6 +993,13 @@ ops(void *arg)
     truncate_op = mmrand(&tinfo->data_rnd, 100, 10 * WT_THOUSAND);
 
     for (intxn = false; !tinfo->quit;) {
+        if (GV(OPS_THROTTLE)) {
+            /* Sleep first to avoid the burst when all threads start. */
+            /* Calculate max delay so that per-table ops/sec is as set. */
```

Comment:
```suggestion
            /* Calculate max delay so that per-table ops/sec is as set. We use 2* here as our random operation uses a flat distribution and the average delay will come out to OPS_THROTTLE_SLEEP_US */
```

Reply:
Done